src/MongoDB.Driver.Core/Core/Operations/ChangeStreamOperation.cs

```diff
@@ -74,6 +66,17 @@ public interface IChangeStreamOperation<TResult> : IReadOperation<IAsyncCursor<T
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>A Task whose result is a cursor.</returns>
         Task<IAsyncCursor<RawBsonDocument>> ResumeAsync(IReadBinding binding, CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Returns a new instance of IChangeStreamOperation with resume options.
+        /// </summary>
+        /// <param name="startAfter">The start after value.</param>
+        /// <param name="resumeAfter">The resume after value.</param>
+        /// <param name="startAtOperationTime">The start at operation time value.</param>
+        /// <returns>
+        /// A new instance of IChangeStreamOperation.
+        /// </returns>
+        IChangeStreamOperation<TResult> WithResumeOptions(BsonDocument startAfter, BsonDocument resumeAfter, BsonTimestamp startAtOperationTime);
```

Comment:
Remove this WithResumeOptions method and replace it instead with the following property that can be used along with ResumeAfter and StartAtOperationTime to reconfigure the resume values of the existing operation:

    /// <summary>
    /// Gets or sets the start after value.
    /// </summary>
    /// <value>
    /// The start after value.
    /// </value>
    BsonDocument StartAfter { get; set; }

Reply:
Done